packages/react-examples/src/react/Nav/docs/NavBestPractices.md

```diff
@@ -9,3 +9,7 @@
 - Use the word that feels right for the navigation. For example, some items may make more sense as nouns (e.g. “Files”), others as adjectives (“Shared”). Use what makes sense for customers, and keep it short!
 - Try to keep your app’s nav in a consistent order across platforms. This sort of consistency increases predictability which drives customer confidence, thus retaining and engaging them.
 - If using a menu button to expand and collapse the `Nav`, use the tooltip “Expand navigation” or “Collapse navigation”.
+
+### Accessibility
+
+If your Nav items lose meaning by being truncated, they might benefit from being styled to wrap their text instead. Although the full text is exposed in a tooltip, that tooltip is not accessible through the keyboard, touch, voice control, or other assistive tech without hover functionality.
```

Comment:
You might add an explicit reference to the example name that shows the code for how to set this up